.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+sudo: false
+node_js:
+    - "4.1"
+
+before_script:
+  #- npm install
+  - env
+
+script:
+  - npm run build
```

Comment:
Can we do blocks of scripts? 
Can we run scripts in parallel inside those blocks?